tests/conftest.py

```diff
@@ -455,20 +455,34 @@ def uvm_with_initrd(
     yield uvm
 
 
-def uvm_booted(microvm_factory, guest_kernel, rootfs, cpu_template):
+@pytest.fixture
+def vcpu_count():
+    """Return default vcpu_count. Use indirect parametrization to override."""
+    return 2
```

Comment:
out of curiosity, how do we override this?

Reply:
For example:

```
@pytest.mark.parametrize("vcpu_count", [1, 2, 7], indirect=True)
def test_new_test(uvm_any):
    pass
```